ui/packages/catalog-ui-search/src/main/webapp/component/filter-builder/filter-builder.view.js

```diff
@@ -183,6 +183,9 @@ module.exports = Marionette.LayoutView.extend({
     this.filterOperator.currentView.turnOffEditing()
     this.filterContents.currentView.turnOffEditing()
   },
+  turnOffNesting: function() {
```

Comment:
Looks like this isn't used anywhere. Is it still needed?

Reply:
its currently being used here https://github.com/codice/ddf/pull/6202/files#diff-1c0b76b4e9e8b174dea0eae50b692807L63